docker-compose/docker-whisk-controller.env

```diff
@@ -30,3 +30,9 @@ INVOKER_INSTANCES=1
 CONTROLLER_INSTANCES=1
 WHISK_SYSTEM_KEY=whisk.system
 RUNTIMES_MANIFEST={"runtimes":{"nodejs":[{"kind":"nodejs","image":{"name":"nodejsaction"},"deprecated":true},{"kind":"nodejs:6","default":true,"image":{"name":"nodejs6action"}}],"python":[{"kind":"python","image":{"name":"python2action"}},{"kind":"python:2","default":true,"image":{"name":"python2action"}},{"kind":"python:3","image":{"name":"python3action"}}],"swift":[{"kind":"swift","image":{"name":"swiftaction"},"deprecated":true},{"kind":"swift:3","default":true,"image":{"name":"swift3action"}}],"java":[{"kind":"java","attached":{"attachmentName":"jarfile","attachmentType":"application/java-archive"},"sentinelledLogs":false,"requireMain":true,"image":{"name":"java8action"},"default":true}]}}
+
+CONTROLLER_LOCALBOOKKEEPING=true
+AKKA_CLUSTER_HOST=localhost
```

Comment:
will this work with docker-machine? It may not matter for now (if seed nodes is empty), but I'm not sure

Reply:
currently docker-compose doesn't work on docker-machine.

Reply:
I mean even w/o this PR, docker-compose was not working on docker-machine.

Reply:
ok